### PR TITLE
[WIP] [NOTEST] Simple list test utility

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

File: scripts/list_tests.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+e.g. list_test . provision
+e.g. list_test file.py
+"""
+    sys.exit(1)
+
+
+def parser(filename, exp=None):
+    try:
+        f = open(filename, 'r')
```
Comment (mfalesni) [inline scripts/list_tests.py]: `with`

**Comment by dajoRH** [comment]
Lint report for commit 7b57fbf0191238b59d860de248c8071cc22e9f47:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: scripts/list_tests.py (hunk 1 above)
Comment (psav) [inline scripts/list_tests.py]: yeh...will do...old habits

**Comment by dajoRH** [comment]
Lint report for commit ac6451e7e2a700292d8ea72b361ebb24ff698615:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**